### PR TITLE
Add FunctionNameFromPath()

### PR DESCRIPTION
**Pull request by dturanski**

### CODE REVIEW

File: pkg/functions/functions.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+package functions
+
+import (
+	"path/filepath"
+	"github.com/projectriff/riff-cli/pkg/osutils"
+)
+
+func FunctionNameFromPath(path string) string {
+	if osutils.IsDirectory(path) {
+		return filepath.Base(path)
```
Comment (trisberg) [inline pkg/functions/functions.go]: what happens if path is `.`

Comment (trisberg) [reply pkg/functions/functions.go]: using `filepath.Abs(path)` should get an absolute path even from a `.`

Comment (dturanski) [reply pkg/functions/functions.go]: Agree

**Comment by trisberg** [comment]
LGTM

**Comment by trisberg** [comment]
Squashed and merged as commit 11ecc8bdf30737a4e3976307ef4a4a2fab7e88d9

**Pull request closed without merging**